src/hydra_zen/structured_configs/_implementations.py

```diff
@@ -485,32 +515,94 @@ def create_just_if_needed(value: _T) -> Union[_T, Type[Just]]:
     return value
 
 
-def sanitized_default_value(value: Any) -> Field:
-    if isinstance(value, _utils.KNOWN_MUTABLE_TYPES):
-        return cast(Field, mutable_value(value))
+def sanitize_collection(x: _T) -> _T:
+    """Pass contents of lists, tuples, or dicts through sanitized_default_values"""
+    type_x = type(x)
+    if type_x in {list, tuple}:
+        return type_x(sanitized_default_value(_x) for _x in x)  # type: ignore
+    elif type_x is dict:
+        return {
+            sanitized_default_value(k): sanitized_default_value(v)
+            for k, v in x.items()  # type: ignore
+        }
+    else:
+        # pass-through
+        return x
+
+
+def sanitized_default_value(
+    value: Any,
+    allow_zen_conversion: bool = True,
+    *,
+    error_prefix: str = "",
+    field_name: str = "",
+) -> Any:
+    value = sanitize_collection(value)
     resolved_value = create_just_if_needed(value)
-    return (
-        _utils.field(default=value)
-        if value is resolved_value
-        else _utils.field(default=resolved_value)
+
+    type_value = type(value)
+
+    # we don't use isinstance because we don't permit subclasses of supported
+    # primitives
+    if allow_zen_conversion and type_value in ZEN_SUPPORTED_PRIMITIVES:
+        type_ = type(resolved_value)
+        conversion_fn = ZEN_VALUE_CONVERSION.get(type_)
+
+        if conversion_fn is not None:
+            return conversion_fn(resolved_value)
+
+    if type_value in HYDRA_SUPPORTED_PRIMITIVES or is_dataclass(resolved_value):
```

Comment:
We need to add support for:
- subclasses of Enums
- omegaconf objects

Reply:
Oh Enums, that's a good one as we have an Enum class in our radar sim.  I'll look through my code and see if there any other good primitives.